### PR TITLE
feature/PROJ-198: implement mobile Settings component (Profile Settings / Account Management, Profile Settings / My payments)

### PR DESCRIPTION
**Pull request by AramZakaryan**
# Overview

This PR implements a redesigned **Account management** and **My payments** tabs of **Settings** component (**Profile Settings**), along with updates to other components that have been influenced by this redesign, all optimized for mobile view.

# Changes

- Redesign of  **MyPayments**, **MyPaymentsList** components, including their styles
- Add new compontent for mobile view: **MyPaymentsCardsMobile** and **PaymentCard** components, including their styles
- Update of styles of the **PaymentMethodSelector** and **AccountManagements** components

Screenshot 2024-11-19 at 11 31 58

Screenshot 2024-11-19 at 11 32 27

### CODE REVIEW

File: src/features/profile/ui/settings/myPayments/myPaymentsCardsMobile/myPaymentsCardsMobile.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { DetailedPayment } from '@/features'
+import { PaymentCard } from '@/features/profile/ui/settings/myPayments/myPaymentsCardsMobile/paymentCard'
```
Comment (EvgeniyaYakunina) [inline src/features/profile/ui/settings/myPayments/myPaymentsCardsMobile/myPaymentsCardsMobile.tsx:2]: Сократи этот импорт. Не хватает экспорта в индекс файле папки myPayments

File: src/features/profile/ui/settings/myPayments/myPayments.tsx [hunk 2]
```diff
@@ -6,8 +6,11 @@ import { Pagination, Select, SelectContainer, SelectItem } from '@photo-fiesta/u
 
 import styles from './myPayments.module.scss'
 
+import { MyPaymentsCardsMobile } from './myPaymentsCardsMobile'
```
Comment (EvgeniyaYakunina) [inline src/features/profile/ui/settings/myPayments/myPayments.tsx:9]: Проверь этот импорт, пожалуйста. Вроде бы должен быть '@/features'

**Review by Lelik7777 — APPROVED** [review]

**Comment by Lelik7777** [comment]
run build production

File: src/features/profile/ui/settings/myPayments/myPaymentsCardsMobile/myPaymentsCardsMobile.tsx (hunk 1 above)
Comment (AramZakaryan) [reply src/features/profile/ui/settings/myPayments/myPaymentsCardsMobile/myPaymentsCardsMobile.tsx:2]: сократил, добавил экспорт

File: src/features/profile/ui/settings/myPayments/myPayments.tsx (hunk 2 above)
Comment (AramZakaryan) [reply src/features/profile/ui/settings/myPayments/myPayments.tsx:9]: поменял на импорт из '@/features'

**Pull request merged**